EKF/ekf_helper.cpp

```diff
@@ -1748,6 +1750,15 @@ float Ekf::kahanSummation(float sum_previous, float input, float &accumulator) c
 	return t;
 }
 
+bool Ekf::isTimedOut(uint64_t last_sensor_timestamp, uint64_t timeout_period)
+{
+	return last_sensor_timestamp + timeout_period < _time_last_imu;
```

Comment:
Can you make sure this (and below) gets inlined by moving the definition to the declaration?

Reply:
yes, done